packages/server/modules/emails/tests/verifications.spec.ts

```diff
@@ -23,7 +23,7 @@ import { EmailSendingServiceMock } from '@/test/mocks/global'
 const mailerMock = EmailSendingServiceMock
 
 const cleanup = async () => {
-  await truncateTables([Users.name, EmailVerifications.name])
+  await truncateTables([Users.name, EmailVerifications.name, 'user_emails'])
```

Comment:
No magic strings please, create a DB helper in dbschema.ts or at the very least stored this as a constant in `domain` or wherever

Reply:
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#5a51bc8fa7034a9bb5b610bbdba5c847